modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/info/item/helper/LayoutInfoItemFieldValuesUpdaterHelper.java

```diff
@@ -52,17 +53,18 @@ public LayoutInfoItemFieldValuesUpdaterHelper(
 	}
 
 	public Layout updateFromInfoItemFieldValues(
-		Layout layout, InfoItemFieldValues infoItemFieldValues) {
+		Layout layout, InfoItemFieldValues infoItemFieldValues,
```

Comment:
I do not understand because we modified this. We are always calling this method with the SegmentsExperienceConstants.ID_DEFAULT

@adolfopa  can you give me more info?

Thanks!